### ISSUE TITLE
Need Static Page

### ISSUE THREAD

**Issue by KaminKevCrew** [michaeltorres1/WiRR#9]
I need to build a static page so that we can have somethin if the back end goes down.

**Comment by michaeltorres1** [comment]
This is a good idea I think we should do this after we get the main features up and running

**Comment by Hakeemmidan** [comment]
I like this idea. Safety is always good. Perhaps we can utilize React Router to implement this.

**Comment by michaeltorres1** [comment]
I agree that using the `React Router` as @Hakeemmidan mentioned is a viable solution to solve this issue. However, being that this is not part of one of MVP's we should mark this issue as an extra feature.

I went ahead and updated the labels to include `extra feature` for this issue.
